### PR TITLE
update Metropolis footer link

### PR DESCRIPTION
**Pull request by jiawendh**

### CODE REVIEW

File: src/components/Metropolis/components/Footer.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ const Footer = () => {
     return (
         <footer className="max-w-screen-xl mx-auto px-5 pt-1 bg-[#131C25] md:flex md:items-center md:flex-wrap">
             <div className="w-full md:w-auto md:pl-12 md:pb-8 flex items-center justify-center md:justify-start">
-                <a href="/" className='hover:no-underline'>
+                <a href="/metropolis" className='hover:no-underline'>
```
Comment (chiew256) [inline src/components/Metropolis/components/Footer.tsx:55]: why navigate to the page itself?

File: static/img/metropolis-api/dexscreener.svg
Comment (chiew256) [inline static/img/metropolis-api/dexscreener.svg:1]: Just consistent to export all as `.png`. If svg, we don't need to put as assets, can put under `.tsx` file extension. Same goes to the `solana.svg`

**Review by chiew256 — COMMENTED** [review]
image


under this breakpoint, can we make it horizontally center as well?

File: src/components/Metropolis/components/Footer.tsx (hunk 1 above)
Comment (jiawendh) [reply src/components/Metropolis/components/Footer.tsx:55]: hmm where should I link the logo to?

**Review by chiew256 — APPROVED** [review]

File: src/components/Metropolis/components/Header.tsx [hunk 2]
```diff
@@ -131,23 +114,15 @@ const Header = () => {
...
-                                        </a>
-                                    );
-                                }
+                                return (
+                                    <a href={link.path} key={link.label} target="_blank" className="text-sm font-semibold py-[0.22rem] text-white transition-colors duration-200 ease-in hover:no-underline hover:text-[#c7f284]">
+                                        {link.label}
+                                    </a>
+                                )
                             })}
+                            <a href={DOCS_URL} target="_blank" className="text-sm font-semibold text-center text-[#c7f284] bg-[#4A5C41]/90 border border-solid border-transparent rounded-full py-[0.6rem] px-[0.93rem] mt-4 transition-colors duration-200 ease-out hover:no-underline hover:text-[#c7f284] hover:border-[#c7f284]">
```
Comment (worlddlckgh) [inline src/components/Metropolis/components/Header.tsx:123]: these classes are extremely hard to read
would prefer splitting them by section

`[0.6rem] px-[0.93rem]`
and these wont work 
0.93*16 = 14.88px, you can't do .88px

**Review by worlddlckgh — APPROVED** [review]

**Pull request merged**